crates/biome_service/tests/invalid/formatter_extraneous_field.json.snap

```diff
@@ -18,7 +18,6 @@ formatter_extraneous_field.json:3:3 deserialize ━━━━━━━━━━
   - enabled
   - formatWithErrors
   - indentStyle
-  - indentSize
```

Comment:
I let the macro filter out deprecated keys, since even though they're technically accepted, I don't think they're a very useful suggestion to the user.